cmake/Findquatlib.cmake

```diff
@@ -65,19 +65,35 @@ else()
 		/usr/local)
 
 	# Look for the library.
-	find_library(QUATLIB_LIBRARY
+	find_library(QUATLIB_LIBRARY_RELEASE
 		NAMES
 		quat.lib
 		libquat.a
-		HINTS
+		PATH_SUFFIXES
+		ReleaseAcademicEdition
+		Release
+		${_libsuffixes}
+		PATHS
 		"${QUATLIB_ROOT_DIR}"
+		"${_progfiles}/VRPN"
+		"${_progfiles}/quatlib"
+		C:/usr/local
+		/usr/local)
+	find_library(QUATLIB_LIBRARY_DEBUG
+		NAMES
+		quat.lib
+		libquat.a
 		PATH_SUFFIXES
+		DebugAcademicEdition
+		Debug
```

Comment:
but it's correct here to only have Debug - don't want RelWithDebInfo in there, or strange things may happen when you try to link using MSVC. (The debug C runtime is different, so it's best not to mix them - in some cases, it's entirely forbidden to mix them)